src/index.ts

```diff
@@ -211,7 +212,7 @@ export default class OtaClient {
         }
         const timestamp = await this.getManifestTimestamp();
         url += `?timestamp=${timestamp}`;
-        return this.httpClient.get(url);
+        return this.httpClient.get<Record<string, any>>(url).catch(() => null);
```

Comment:
Well I did not used `Record` keyword that often to judge but according to [docs](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) this is not the right usage. We are returning here the content of the file. Which can be either plain string or object (for json files) or binary. Or`null` in case of error. So to me the return type of this function is `string | object | null`.

Reply:
I will add the string type, forgot not all files are json

Reply:
Can you also please update the `content` field type in `LanguageTranslations` to make it clear that it can contain `null`

Reply:
Done!

Reply:
And also not only `string` or `Repord` but also a binary type can be returned. Which can be a either a `Buffer` for Node or `ArrayBuffer` for Browser.

Reply:
heh, now I recall why I decided to go with `any` :(